requirements.txt

```diff
@@ -163,13 +166,14 @@ soupsieve==1.9.3
 SQLAlchemy==1.3.8
 sshpubkeys==3.1.0
 stevedore==1.31.0
-stringscore==0.1.0
+#stringscore==0.1.0
```

Comment:
What issue did you have on this one? It seemed to install fine on my end.

Reply:
Failed on centos8. Can look at why in another PR.